i18n/ru.po

```diff
@@ -3378,10 +3397,15 @@ msgid ""
 "Middle Mouse: \tClick and drag to zoom in or out.\n"
 "Right Mouse: \tClick and drag to move the view.\n"
 msgstr ""
+Инструмент изменения обзора\n"
```

Comment:
Is this correct? It looks like there are un-matching " here.

Reply:
Yes, that's a mistake. I've fixed in all three cases, thank you